crates/ruff/src/rules/pyupgrade/rules/f_strings.rs

```diff
@@ -420,7 +422,19 @@ pub(crate) fn f_strings(
     }
 
     let mut diagnostic = Diagnostic::new(FString, expr.range());
-    if checker.patch(diagnostic.kind.rule()) {
+
+    // Avoid autofix if there are comments within the call:
+    // ```
+    // "{}".format(
+    //     0,  # 0
+    // )
+    // ```
+    if checker.patch(diagnostic.kind.rule())
+        && !checker
+            .indexer()
+            .comment_ranges()
+            .intersects(TextRange::new(attr.end(), expr.end()))
+    {
```

Comment:
Alternatively we could downgrade from a `suggested` to a `manual` fix if we detect comments?

Reply:
(Although this change is probably best for right now)